image.go

```diff
@@ -20,6 +20,8 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
+	// Image provides access to internal image info
```

Comment:
@Random-Liu @ijc Checking if an image is ready is fairly cheap. Do you want a helper function to perform this?

As far as indexing goes, the "image id" is really a docker concept (and an odd one at that). It is possible to put a label on with the docker image id, but it would make a lot more sense to just start using the digests.